readme.md

```diff
@@ -11,47 +11,70 @@ Utopia is an integrated design and development environment for React. It uses Re
-# For contributors: Installing Utopia on your machine
 
-Utopia is browser-based. To run it locally, clone the repo, and then set up the server and webpack by following these instructions.
 
-⚠️ We're looking into M1 related issues for some folks, but have it running successfully on three machine
+### Contributing and Bug Reporting
 
-## Prerequisites
+* Have a bug to report? File an issue, or find us on [Discord](https://discord.gg/NEEnPKCgzC)
+* Want to contribute? Please read our [contributing](contributing.md) doc to get started, and join our Discord. It's a big project and the core contributors are  there to help.
 
-- **If using Windows** you'll first need to set up the [Windows Subsystem for Linux (wsl)](https://docs.microsoft.com/en-us/windows/wsl/install-win10). All following steps and commands will assume you are using the wsl.
-- On **macOS** you need [brew](https://brew.sh/) and must run `brew install gmp` first. On new M1 Macbooks please follow [these steps](https://github.com/concrete-utopia/utopia#m1-macbook) to install brew and run the server the first time.
-- [nix-shell](https://nixos.org/download.html). If you are on macOS Catalina or later, you will be prompted to include an extra flag in the install script. If using Windows follow [this guide](https://nathan.gs/2019/04/12/nix-on-windows/). If you don't want to use nix, we have instructions [here](https://github.com/concrete-utopia/utopia#running-this-without-nix)
-- Recommended: [direnv](https://github.com/concrete-utopia/utopia#using-direnv-to-make-your-life-easier). If you don't have `direnv` installed, you'll need to run `nix-shell` before any of the `start` commands, and switching to nix will be a bit slower.
 
-## Contributing and Bug Reporting
 
-We welcome contributions. Utopia is a big project, but we're here to help, and are happy to pair up with you. The easiest way is to file an issue, or reach out on [Discord](https://discord.gg/NEEnPKCgzC). Please read our [contributing](contributing.md) doc to get started
+# Install Utopia locally
+Just want to _use_ Utopia? Head to [utopia.app/project](https://utopia.app/project) instead.
 
-# Run the Editor
+Utopia is browser-based. To run it locally, clone the repo, install the prerequisites, and run the editor.
 
-The first time running the editor, run the following script:
+
+
+### Install Prerequisites
+
+- (Windows only) Install WSL (Windows only)
+- (MacOS only: Install [brew](https://brew.sh/) and run run `brew install gmp`. On an M1 Mac, follow [these steps](https://github.com/concrete-utopia/utopia#m1-macbook)
+- Install [nix-shell](https://nixos.org/download.html). On Windows? Follow [this guide](https://nathan.gs/2019/04/12/nix-on-windows/). 
+- Recommended: Install [direnv](https://github.com/concrete-utopia/utopia#using-direnv-to-make-your-life-easier). 
+
+### Run the Editor for the first time
+
+Install the prerequsites, then run this in your terminal. NB There's no docker image at the moment.
 
 ```
+nix-shell
```

Comment:
You can make this `nix-shell --run start-full` and it'll run that within the shell once the dependencies have been pulled.